squarelet/organizations/serializers.py

```diff
@@ -190,6 +190,7 @@ class Meta:
             "created_at": {"read_only": True},
             "accepted_at": {"read_only": True},
             "rejected_at": {"read_only": True},
+            "uuid": {"read_only": True},
```

Comment:
This should not be added

Reply:
I added this id for the invitation to the data returned here because I didn't see another way of getting it. Our frontend needs the invitation ID in order to accept or reject it - it calls the API's `/pp-api/invitations/{uuid}` endpoint to do so. I've removed this for now, but I'm hoping we can chat about how best to accomplish this @TylerFisher, thanks!

Reply:
I removed this line.

Reply:
Note that after our discussion about IDs earlier today that I've added this back :)